components/patient-data/impl/src/main/java/org/phenotips/data/internal/PhenoTipsGene.java

```diff
@@ -86,7 +86,7 @@
 
     /**
      * The gene status, one of possible values: "candidate" (default value), "rejected_candidate", "rejected", "solved",
-     * "carrier".
+     * "carrier", "candidate_novel_disease", "candidate_novel_phen", "umc", "umc_vus", "umc_msv".
```

Comment:
Comments should not list the available statuses, because they may change. Instead, a link to where the statuses are defined should be provided.